src/components/KYJobListing/styles/JobListingCard.styled.ts

```diff
@@ -0,0 +1,80 @@
+import styled from 'styled-components'
+
+//media: set flex-direction to col
+export const Card = styled.div<{ active?: boolean }>`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  height: 140px;
+  margin: 30px;
+  padding: 20px;
+  background-color: white;
+  border-radius: 5px;
+  box-shadow: 5px 10px 10px #dedede;
+  ${(props) =>
+    props.active &&
+    `
+    border-left: 5px solid hsl(180, 29%, 50%);
+  `}
```

Comment:
the formatting is a bit weird, they can all be put in the same line in this case

Reply:
resolved in commit e616b1a